boards/sparkfun/artemis_nano/docs/index.rst

```diff
@@ -0,0 +1,88 @@
+.. _artemis_nano:
+
+Sparkfun Artemis Nano
+######################
+
+The SparkFun Artemis Nano is a versatile and user-friendly development board that utilizes
+ Ambiq's ultra-low power Apollo3 Blue SoC.
+
+.. image:: ./sparkfun_artemis_nano.png
+   :align: center
+   :alt: Artemis Nano
+
+Hardware
+********
+
+- Apollo3 Blue SoC with up to 96 MHz operating frequency
+- ARM® Cortex® M4F core
+- 16 kB 2-way Associative/Direct-Mapped Cache per core
+- Up to 1 MB of flash memory for code/data
+- Up to 384 KB of low leakage / low power RAM for code/data
+- Integrated Bluetooth 5 Low-energy controller
+
+For more information about the SparkFun RedBoard Artemis Nano:
+
+- `SparkFun RedBoard Artemis Nano`_
+
+Supported Features
+==================
+
+The Sparkfun Redboard Artemis Nano board configuration supports the following hardware features:
```

Comment:
use the board name for west flash with \`\` around it and call it board target, not configuration

Reply:
Done